### ISSUE TITLE
Unable to watch Wizja.TV on KODI v 17.1

### ISSUE THREAD

**Issue by IzabellaC** [mrknow/filmkodi#870]
## Please follow the guide below 
- Put an "x" into all boxes [ ] relevant to your *issue*
- Use *Preview* tab to see how your issue will actually look like
---
## Kodi & Addon 
- Kodi verison: _17.1 
- [ ] Filmy online - Mrknow
- [ X] MrknowTV
- [ ] Specto
- [ ] FanFILM
- Version / Wersja: Latest
---
## Descryption / Opis
Unable to watch WIzja.TV Premium on Kodi 17.1. The issue started today.
Mam wersje Premium Wizja.TV i od dzis nie mam dostepu do kanalow na Wizja.TV poprzez program Kodi wersja 17.1. Problem zaczal sie dzisiaj. 

## Donation / Darowizna
This addons are free, but you can always thank the author for ... / Te wtyczki są darmowe, ale zawsze możesz podziękować autorowi za jej stworzenie...

[![paypal](https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif)](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=HEV6TJY83S3Q4&lc=PL&item_name=Donation-Darowizna&currency_code=PLN&bn=PP%2dDonationsBF%3abtn_donateCC_LG%2egif%3aNonHosted)

**Comment by Arek76** [comment]
troszke cierpliwosci i wszystko dziala:) Czy nie ?

**Comment by IzabellaC** [comment]
Tak, zaczelo 30 minut temu. Dzieki!!!!!! Na czym polegal problem?

**Comment by frijerz** [comment]
Wtyczka Wizja nadal nie działa.

**Comment by IzabellaC** [comment]
Dzien Dobry
Poniownie nie widze kanalow Wizja.TV na KODI 17.1. Czy mozecie
Panowie/Panie sprawdzic i przywrocic te kanaly?

**Comment by janpawel62** [comment]
potwierdzam, na Kodi 17.1 pokazuje mi moje konto premium, ale po kliknięciu w Telewizja online nie wyświetla się lista kanałów :(

**Comment by IzabellaC** [comment]
Tak, ale teraz juz dziala. Super! Dzieki!!!!!!!

2017-05-12 10:23 GMT-04:00 janpawel62 <notifications@github.com>:

**Comment by kobra7950gt** [comment]
Wizja.tv nie działa

**Comment by janpawel62** [comment]
IzabellaC może u ciebie działa, ale u mnie na Win 7 Kodi 17.1 nie działa, mimo iż jeszcze 2 tygodnie było wszystko ok, wygląda na to że zależy to od wersji Kodi i systemu na którym zainstalowana jest wtyczka. Najgorsze jest to że wykupiłem Premium na pół roku i nie mogę oglądać teraz (po jakiś zmianach) TV :(

**Comment by kobra7950gt** [comment]
kodi 17.1 wtyczka wizja nie działa POMOCY!!

**Comment by Arek76** [comment]
U mnie i u moich klientow dziala choc uzywaja roznych wersji kodi i roznych
wersji androida. Byl problem dzis ale wszystko wrocilo do normy , moze
zaktualizujcie wtyczke

12 maj 2017 18:55 "kobra7950gt" <notifications@github.com> napisał(a):

kodi 17.1 wtyczka wizja nie działa POMOCY!!

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/mrknow/filmkodi/issues/870#issuecomment-301129837>, or mute
the thread

**Comment by janpawel62** [comment]
usystematyzuję:
wtyczka nie działa na systemie WINDOWS 7 w KODI 16.1 i 17.1
Na Androidzie teraz sprawdziłem i jest OK
Wcześniej Wizja.tv była dostępna we wtyczce MrknowTV, teraz jej tam nie ma a w oddzielnie pobranej wtyczce Wizja.tv [http://wizja.tv] Mrknow nie otwiera się lista kanałów , ale za to wyświetla się informacja o czasie obowiązywania konta premium. W razie czego mogę podesłać jakiś filmik ze zrzutu ekranu...

**Comment by janpawel62** [comment]
wiadomość z dnia dzisiejszego: nagle wyświetliła mi się lista kanałów!
Nie wiem czy coś się zmieniło we wtyczce, ale... odpaliłem dziś ccleaner  by posprzątać w systemie i może to pomogło :)

**Comment by IzabellaC** [comment]
Nie jestem w stained otworzyc w KODI zadnych kanalow. Prosze o pomoc
